src/views/add_event.hbs

```diff
@@ -0,0 +1,29 @@
+<div class="">
+  <form action="/insertEvent" method="post">
+      <label for="title-event">Title:</label>
+      <input type="text" name="title" class="pa2 bg-light-gray" placeholder="Name of the event?" id="title-event" required>
+
+      <label for="day-input" class="">Day:</label>
+      <input type="number" name="day" min="1" max="31" value="1" class="bg-light-gray" id="day-input" required>
```

Comment:
I think it will be better if we change the type to text..since it's not a drop down list and the user can still type the input if he wants

Reply:
but then the user would be able to input stings and words which would result in an error